scripts/check-delegates-test.js

```diff
@@ -0,0 +1,19 @@
+import { strict as assert } from 'node:assert';
+import { checkDelegates } from './check-delegates.js';
+
+const lex = `Rob Palmer (RPR)\nUjjwal Sharma (USA)\nChris de Almeida (CDA)`;
+const missing = `Chris de Almeida (CDA)\nRob Palmer\nUjjwal Sharma (USA)`;
+const uppercaseLatin = `Chris de Almeida (CDA)\nRob Palmer (竄)\nUjjwal Sharma (USA)`;
+const twoLetter = `Chris de Almeida (CDA)\nRob Palmer (RP)\nUjjwal Sharma (USA)`;
+const threeLetter = `Chris de Almeida (CDA)\nRob Palmer (ROBPALMER)\nUjjwal Sharma (USA)`;
+const duplicate = `Chris de Almeida (CDA)\nRob Palmer (RPR)\nUjjwal Sharma (USA)\nUjjwal Sharma (USA)`;
+const valid = `Chris de Almeida (CDA)\nRob Palmer (RPR)\nUjjwal Sharma (USA)`;
+
+assert.throws(()=>checkDelegates(lex), { message: /Not in lexicographic order/});
+assert.throws(()=>checkDelegates(missing), { message: /Missing abbreviation for/});
+assert.throws(()=>checkDelegates(uppercaseLatin), { message: /Abbreviations must be all uppercase Latin letters/});
+assert.throws(()=>checkDelegates(twoLetter), { message: /not in allowlist. New delegate abbreviations must be three letters/});
+assert.throws(()=>checkDelegates(threeLetter), { message: /New delegate abbreviations must be three letters/});
```

Comment:
```suggestion
assert.throws(() => checkDelegates(threeLetter), { message: /New delegate abbreviations must be three letters/ });
```
and similar throughout

Reply:
if we are enforcing styles, we should set it up via eslint/prettier

Reply:
i mean sure, but this is pretty universally idiomatic spacing

Reply:
@ctcpip That's not how this works lol. Just because we don't have CI style enforcement yet doesn't mean we have no style rules. Before widespread use of linters, it wasn't just a wild West.

Once we have a linter in CI, if someone asks you for further style changes that are not yet enforced by the linter, *then* you can respond "well the linter doesn't enforce it, so we should update the linter".

Reply:
@michaelficarra Understood. For context, Jordan and I have collaborated on numerous PRs where we have style/lint changes and a recurring theme is that we don't have linting set up, so we go and set it up.

I don't disagree with the convention.

Reply:
done

Reply:
just noticed the white space reco for the curly -- fixed